src/igvfd/schemas/differentiated_cell.json

```diff
@@ -57,7 +57,7 @@
             "title": "Post-differentiation time",
             "description": "The time that elapsed past the point when biosamples were differentiated.",
```

Comment:
you forgot to change default schema version to be 5

Reply:
Done, missed from re-basing.